test/model/src/driver/kv_workload_generator.cpp

```diff
@@ -313,6 +326,26 @@ kv_workload_generator::choose_table(kv_workload_sequence_ptr txn)
     return _tables_list[_random.next_index(_tables_list.size())];
 }
 
+/*
+ * kv_workload_generator::generate_rand_stress_configs --
+ *     Generate random WiredTiger timing stress configurations.
+ */
+std::string
+kv_workload_generator::generate_rand_stress_configs()
```

Comment:
How about adding `timing` in the function name for more clarity?

Reply:
Fixed in b15a4f491906d1b92d4955cb07a321633d9f7722